advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeType.java

```diff
@@ -152,53 +155,118 @@
 		NAME_RESOURCE_FILE = pName + ".types"; //$NON-NLS-1$
 	}
 
+	/** FactoryMap collection.
+	 *
+	 */
+	private static final Map<AttributeType, StringFactory> FACTORY_MAP =
```

Comment:
I will make  a note of it now and leave it for a future PR, I promise.  I am a bit behind my targets this week. This PR has taken quite a bit of my time. I will now try to look for similar issues.